client/app/bundles/course/assessment/submission/components/answers/index.tsx

```diff
@@ -146,6 +152,15 @@ const SubmissionAnswer = <T extends keyof typeof QuestionType>(
     },
   };
 
+  const ErrorComponentMap = {
```

Comment:
Create a custom hook that handles the type of the error and returns the translation.

Reply:
done